zebra-network/src/protocol/external/message.rs

```diff
@@ -340,20 +340,29 @@ pub enum Message {
     FilterClear,
 }
 
+/// The maximum size of the rejection message
+const MAX_REJECT_MESSAGE_LENGTH: usize = 12;
+/// The maximum size of the rejection reason
+const MAX_REJECT_REASON_LENGTH: usize = 100;
```

Comment:
From https://github.com/zcash/zcash/blob/1b5ab4a06e52c23aca48afb1ebb72a25ef5dd3ca/src/main.cpp#L7334-L7335 we know zcashd uses `MAX_REJECT_MESSAGE_LENGTH` for the message field. This is equal to 111: https://github.com/zcash/zcash/blob/1b5ab4a06e52c23aca48afb1ebb72a25ef5dd3ca/src/main.h#L117

I added `100` here instead of `111` as we want a  slightly smaller value: https://github.com/ZcashFoundation/zebra/issues/4632#issue-1274130910

Reply:
Can you put this explanation in a doc comment?

It will help us understand why we chose `100` when we look at this code later.